code/modules/projectiles/ammunition/caseless.dm

```diff
@@ -0,0 +1,120 @@
+
+// Caseless Ammunition
+
+/obj/item/ammo_casing/caseless
+	desc = "A caseless bullet casing."
+	firing_effect_type = null
+
+/obj/item/ammo_casing/caseless/fire_casing(atom/target, mob/living/user, params, distro, quiet)
```

Comment:
Should probably have same arguments as base one ? Not sure you actually changed anything here.